js/flightlog.js

```diff
@@ -1145,9 +1170,27 @@ FlightLog.prototype.rcCommandRawToThrottle = function(value) {
     return Math.min(Math.max(((value - this.getSysConfig().minthrottle) / (this.getSysConfig().maxthrottle - this.getSysConfig().minthrottle)) * 100.0, 0.0),100.0);
 };
 
-FlightLog.prototype.rcMotorRawToPct = function(value) {
+FlightLog.prototype.rcMotorRawToPctEffective = function(value) {
+
     // Motor displayed as percentage
     return Math.min(Math.max(((value - this.getSysConfig().motorOutput[0]) / (this.getSysConfig().motorOutput[1] - this.getSysConfig().motorOutput[0])) * 100.0, 0.0),100.0);
+
+};
+
+FlightLog.prototype.rcMotorRawToPctPhysical = function(value) {
+
+    // Motor displayed as percentage
+    let motorPct;
+    if (this.getSysConfig().fast_pwm_protocol < FAST_PROTOCOL.indexOf('DSHOT150')) {
```

Comment:
I think it will be better if we extracted the determination if the protocol is Dshot / digital into a function, and used a switch statement like we do in `checkMotorProtocolEnabled()` in `drivers/motor.c` in the firmware.

Reply:
Added similar method. I didn't like too much the switch but it does not hurt neither...